eng/packages/http-client-csharp/eng/scripts/Generation.psm1

```diff
@@ -49,6 +50,10 @@ function Get-TspCommand {
         $command += " --option @azure-typespec/http-client-csharp.api-version=$apiVersion"
     }
 
+    if ($forceNewProject) {
```

Comment:
Why did we need this?

Reply:
if we ever change new project scaffolding the only way that code is triggered is if the csproj doesn't exist, but it will exist so you would need to go delete all csproj for all local and spector projects which is annoying.

This flag allows us to force the generator to run new project scaffolding even if the csproj already exists.